app/lib/screens/history/history.dart

```diff
@@ -11,6 +18,36 @@ class _TravelHistoryState extends State<TravelHistory> {
     print('Init: Travel history');
   }
 
+  final pdf = pw.Document(); //Create a blank document
+  //add data into the pdf file
+  buildPdf() {
+    pdf.addPage(pw.MultiPage(
+      pageFormat: PdfPageFormat.a4,
+      margin: pw.EdgeInsets.all(34),
+      build: (pw.Context context) {
+        return <pw.Widget>[
+          /*
+              We will be calling API and fetching data from the database here.
+              Now just adding random text for experimentation purpose. Once we have databse/API set up, will
+              experimenting on fetching data
+              */
+          pw.Header(level: 0, child: pw.Text("Shipper Information")),
+          pw.Paragraph(text: 'Some Information'),
+          pw.Paragraph(text: 'Some Information'),
+          pw.Header(level: 1, child: pw.Text("Travel Information")),
+          pw.Paragraph(text: 'Some Text 1'),
+          pw.Paragraph(text: 'Some Text 2'),
+        ];
+      },
+    ));
+  }
+
+  Future savePdf() async {
```

Comment:
I see this warning in my IDE upon inspecting your code:
![Screen Shot 2021-01-13 at 7 59 59 PM](https://user-images.githubusercontent.com/32527219/104535117-2b71a900-55db-11eb-9734-772ec4c8a430.png)

- [ ] Please fix or find out why

Reply:
@holtzmak I don't find this warning into my IDE. Where exactly you get this warning? I checked in my problem window I have no error or warnings. I think you need to go to `pubspec.yaml` file and do `pub get`. I think this should resolve the issue.

Reply:
I did not try `flutter pub upgrade`: either my IDE was out of date or this fixed it. No problem here.